examples/insurance/README.md

```diff
@@ -30,6 +39,14 @@ You can finish configuring the flows and run the steps to complete the integrati
 10. Run the `CustomerMaster` flow to master the Advantage and Bedrock customer data. This merges documents for two matching customers in the final database. You can view the results in the Browse Data view.
 
 
+## How to enrich Customer Data using custom step
+
+1. Start the Data Hub and select the project folder: examples/insurance
```

Comment:
I think it'd be better for this to say "Start QuickStart and select..." so that it's clear that the user needs to run QuickStart. 

Probably best to include a note saying that the user will need to install the application as well, which usually requires starting with a clean MarkLogic instance that doesn't have a data hub already installed.